### ISSUE TITLE
Enable dynamic optimization for Windows CI runners

### ISSUE THREAD

**Issue by mknaranja** [SciCompMod/memilio#1014]
In #894, dynamic optimization through automatic differentiation was introduced for a first particular ODE-based model.

However,  dynamic optimization needed to be deactivated in the windows CI. This is due to the setting that, by default, there is no adequate fortran compiler in the Windows runners working with MSVC which would be necessary to build the [MUMPS ](http://mumps-solver.org/index.php?page=home) direct solver.

A corresponding workaround could be to pre-install a MUMPS version with [spack](https://spack.readthedocs.io/en/latest/).

Nevertheless, the implementation in #894 had been tested manually on Windows.

@hannemann-tamas Is this a correct summary? Anything to add?

**Comment by hannemann-tamas** [comment]
This is correct. If I only have to update the CMakeLists.txt of Ipopt. This should not be too difficult.
